### PR TITLE
Fixed some markdown lint warnings and a typo

### PR DESCRIPTION
**Pull request by mippzon**
Fixed some minor markdown lint warnings and a typo in the pull request template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,8 +92,8 @@ You should have a recent version of rust and cargo installed. You don't need nig
 
 This is mostly a note for me on how to release this thing:
 
-- Update versions in `README.md`, `static/index.html`, `Cargo.toml`, `snapcraft.yaml`.
-- `git commit` and `git tag -s`, `git push`.
-- `cargo publish`
-- Releases will automatically be deployed by Travis.
-- Update AUR package.
+-   Update versions in `README.md`, `static/index.html`, `Cargo.toml`, `snapcraft.yaml`.
```
Comment (svenstaro) [inline README.md]: You sure about these? It looks fine as it is.

Comment (svenstaro) [reply README.md]: I just ran this through a linter of my own and I believe it's correct as it is.

Comment (mippzon) [reply README.md]: No problem, then you can close this one again! My markdown linter in Atom registered warnings for those rows, but if yours says it's fine it probably is.

**Comment by svenstaro** [comment]
Alright, I fixed the two issues. Thanks for posting this, though. Closing.

**Pull request closed without merging**